zephyr/CMakeLists.txt

```diff
@@ -868,6 +914,15 @@ zephyr_library_sources_ifdef(CONFIG_COMP_MUX
 	${SOF_AUDIO_PATH}/mux/mux_${ipc_suffix}.c
 )
 
+zephyr_library_sources_ifdef(CONFIG_COMP_MFCC
+	${SOF_AUDIO_PATH}/mfcc/mfcc.c
+	${SOF_AUDIO_PATH}/mfcc/mfcc_setup.c
+	${SOF_AUDIO_PATH}/mfcc/mfcc_common.c
+	${SOF_AUDIO_PATH}/mfcc/mfcc_generic.c
+	${SOF_AUDIO_PATH}/mfcc/mfcc_hifi3.c
+	${SOF_AUDIO_PATH}/mfcc/mfcc_hifi4.c
```

Comment:
Do we need to check HiFi flavor here before we build ?

Reply:
MFCC is  not updated yet similarly as Tim did for other modules. Should do similar maintenance for it, move all to module directory etc. I think this would be a good first step to get it to build and work again. The maintenance should be quite straightforward copy of similar work.